canonicalwebteam/flask_base/request.py

```diff
@@ -0,0 +1,15 @@
+from flask import request, session
+
+
+def set_cache_control_headers(response):
```

Comment:
Couldn't we just do this in app.py? I'm not a huge fan of creating a new file per function.